.ci/install_proxy.sh

```diff
@@ -23,10 +23,23 @@ source "${cidir}/lib.sh"
 
 clone_build_and_install "github.com/clearcontainers/proxy"
 start_proxy_cmd="sudo systemctl start cc-proxy"
+upstart_source_file="${cidir}/data/cc-proxy.conf"
+upstart_destr_file="/etc/init/cc-proxy.conf"
+
+
+# If we are running under the metrics CI system then we do not want the proxy
+# to be dynmaically changing the KSM settings under us - we need control of them
+# ourselves
+if [[ $METRICS_CI ]]; then
+	proxy_systemd_file=$(sudo systemctl show cc-proxy | fgrep FragmentPath | awk 'BEGIN{FS="="}{print $"}')
+	sudo sed -i 's/\(^ExecStart.*$\)/\1 -ksm initial/' "${proxy_systemd_file}"
```

Comment:
Won't this require a `sudo systemctl daemon-reload` after the `sed` (note that Upstart doesn't require any further action)?

Reply:
ah, so it will! I wonder if that happens elsewhere right now - maybe with the prior clone_build_and_install...
This will be the first launch of the proxy I believe, as this is the 'start from scratch every time' CI - so, maybe the $start_proxy_cmd just 'works' - but, I think a daemon reload is still appropriate - I'll go add.